CadRevealRvmProvider/Converters/CapVisibilityHelpers/ComparerHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace CadRevealRvmProvider.Converters.CapVisibilityHelpers;
+
+public static class ComparerHelper
+{
+    public static bool CheckOverlap(
+        bool isCurrentPrimitive,
+        float radius1,
+        float radius2,
+        float radius3,
```

Comment:
can we have better names for these radiuses?

Reply:
Agree 👍 The names should suggest that radius1 is the one that is overlapped by the two others. 

Maybe use a default value for radius 3, since it isn't used in all cases. Or create two methods where one takes 2 radii, while the other takes 3 radii. The one with 3 could call the first one with radius 1 and 2, and then for radius 1 and 3, before determining the resulting boolean.

Reply:
isCurrentPrimitive is a bit hard to read without knowing the context too, btw 🤔 maybe consider renaming it for clarity